packages/services/src/services/Translation/__tests__/Translation.test.js

```diff
@@ -76,7 +98,69 @@ describe('TranslationAPI', () => {
         origin: 'https://ibm.com',
       },
     });
-
+    delete response['id'];
     expect(response).toEqual(responseSuccess);
   });
+
+  it('should return a json with a recent timestamp', async () => {
+    const mockDate = 1546300800000; // Epoch time of January 1, 2019 midnight UTC
```

Comment:
Also please refer to how the ID is set via `Object.assign()`:

```suggestion
    const uniqueId = `__translation_cache__${Math.random().toString(36).slice(2)}`;
    const mockDate = 1546300800000; // Epoch time of January 1, 2019 midnight UTC
```